src/Procuration/Filter/ProcurationProxyProposalFilters.php

```diff
@@ -36,14 +36,28 @@ public function apply(QueryBuilder $qb, string $alias): void
         parent::apply($qb, $alias);
 
         $status = $this->getStatus();
+
+        $qb->andWhere("$alias.disabled = :disabled");
+
         if (self::UNASSOCIATED === $status) {
-            $qb->andWhere("$alias.foundRequest IS NULL");
+            $qb
+                ->andWhere(
+                    $qb->expr()->orX(
+                        "$alias.frenchRequestAvailable != 0",
+                        "$alias.foreignRequestAvailable != 0"
+                    )
+                )
+                ->setParameter('disabled', false)
+             ;
         } elseif (self::ASSOCIATED === $status) {
-            $qb->andWhere("$alias.foundRequest IS NOT NULL");
+            $qb
```

Comment:
PO sayed associated will no longer be a simple associated proxy but a fully associated proxy (no more availability)